pyproject.toml

```
@@ -27,6 +27,10 @@ dependencies = [
     "mnemonic >=0.19",
     "hdwallets >=0.1.0",
     "typing-extensions >=3.7.4; python_version < '3.8'",
+    "requests==2.26.0",
+    "protobuf >= 3.19.1",
+    "google >= 3.0.0",
```

```suggestion
    "protobuf >= 3.19.1",
```

As noted in the review of the previous PR, we shouldn't need `requests` or `google` deps.

---

changed with the two recent commits. Removing requests also involves removing the two helper functions ``broadcast_rpc_tx`` and ``braodcast_api_tx``